docs/DeveloperGuide.md

```diff
@@ -602,29 +602,27 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 | Priority | As a … | I want to … | So that I can…|
 | - | - | - | - |
-| `* * *` | new user | see usage instructions | refer to instructions if I forget how to use the app |
 | `* * *` | confused user | have a help command | learn how to use the application |
-| `* * *` | user | add a new person | |
-| `* * *` | user | add issues | |
-| `* * *` | user | see a list of pending tasks sorted by deadline | prioritise which to do |
-| `* * *` | user | search for tasks | |
+| `* * *` | user | add issues | track associated issues with rooms and their status|
+| `* * *` | user | see a list of pending issues sorted by deadline | prioritise which issue to work on |
+| `* * *` | user | search for issues | find a specific task and get its information or tasks matching a specific criteria|
```

Comment:
Maybe want to break it up ? 
| `* * *` | user | search for issues | find a specific task and get its information |
| `* * *` | user | search for issues | find tasks matching a specific criteria |

Reply:
Good idea, thanks